inflection/src/inflection/dictionary/Inflector.hpp

```diff
@@ -41,6 +41,7 @@ class inflection::dictionary::Inflector
     const Inflector_MMappedDictionary mmappedDictionary;
 
     friend class DictionaryMetaData_MMappedDictionary;
+    friend struct std::default_delete<Inflector>;
```

Comment:
This looks suspicious. What's going on here?

Reply:
Such a declaration is missing the point of making it private in the first place. It's to prevent accidental deletion by non-friends. If std::unique_ptr is to be used to delete it, then it's better to just make the destructor public. It is odd to make a subclass destructor like this, and it's better to just make it public than to add another friend.